src/stacking-group.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import Avatar from './avatar';
+import Image from './image';
+import * as styles from './stacking-group.css';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import classNames from 'classnames';
+
+type dataArrayType = {
+    src?: string;
+    initials?: string;
+};
+
+type StackingGroupProps = {
```

Comment:
If there is only one component in the file, just call this type Props:
```suggestion
type Props = {
```